### PR TITLE
Remove all occurrences of qWait in tests/

### LINKED ISSUE #2236: Eliminate all calls to `qWait` in tests/
Probably all instances of `PyQt5.QtTest.QTest.qWait` are potential race conditions because they are used as a proxy to ensure that something has happened in Qt without actually checking.

In the test suite those should be replaced by using the `qtbot.wait_*` functions from pytest-qt. E.g. asserts that something has happened in Qt should be wrapped in `wait_until` as documented here: https://pytest-qt.readthedocs.io/en/latest/wait_until.html.

### PR DESCRIPTION
**Pull request by matrss**
Fixes #2236.

### CODE REVIEW

File: tests/test_meta.py [hunk 1]
```diff
@@ -37,3 +37,19 @@ def test_processEvents_is_not_used_in_tests(request):
         assert (
             "processEvents" not in test_file.read_text()
         ), "processEvents is mentioned in {}".format(test_file.relative_to(request.config.rootdir))
+
```
Comment (ReimarBauer) [inline tests/test_meta.py:40]: much likes :)

**Review by ReimarBauer — COMMENTED** [review]
I grabbed for self._connect_to_mscolab() 

There are a few not using qtbot,
test_mscolab_version_history 52, test_mscolab_admin_window 65, test_mscolab_merge_waypoints 67, test_mscolab_operations 61

I guess they have always the same problem, should they also become changed?

**Comment by matrss** [comment]
So far I've taken the pragmatic approach of "only fix the tests that actually exhibit issues in practice". There are probably many more places that could fail under the right conditions. That said, I'll change those 4 methods as well, since that's pretty simple.

Maybe in a future refactor those methods could be unified into one, since they all do effectively the same.

**Review by ReimarBauer — APPROVED** [review]
Thx

**Pull request merged**